src/app/app.component.ts

```diff
@@ -23,15 +27,13 @@ export class AppComponent implements OnInit {
 
   ngOnInit(): void {
     const runTime = () => {
-      setTimeout(
-        () => {
-          this.snake.move();
-          if(!this.isGameOver()){
-            runTime();
-          }
-        }, 
-        STEP_TIME
-      );
+      setTimeout(() => {
```

Comment:
I personally prefer the way it was before, but I'm fine with this style as well if you insist on it 😄

Reply:
This change is probably due to a formatter.